lib/components/src/controls/Boolean.tsx

```diff
@@ -36,14 +36,14 @@ const Label = styled.label(({ theme }) => ({
   },
 
   span: {
-    minWidth: 60,
     textAlign: 'center',
     fontSize: theme.typography.size.s1,
     fontWeight: theme.typography.weight.bold,
     lineHeight: '1',
     cursor: 'pointer',
     display: 'inline-block',
-    padding: '8px 16px',
+    padding: '7px 15px',
```

Comment:
All inputs are 28px in height. Reducing the padding makes this control 28px tall as well.